config/tsconfig.test.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    "composite": true,
+    "checkJs": true,
+    "emitDeclarationOnly": false,
```

Comment:
`"emitDeclarationOnly": false,` worth it for our test tsconfig? since it default's to `false`

Reply:
If `noEmit` is `false` then `emitDeclarationOnly` cannot be `true` (TS will error and abort).  At some point, the config this one extended had `emitDeclarationOnly: true` in it, which I needed to override with `false`.  That could be removed now.